### PR TITLE
Restore <body> bottom padding

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

Restore `<body>` bottom padding accidentally removed in getredash/redash#3697

### CODE REVIEW

File: client/app/assets/less/inc/base.less [hunk 1]
```diff
@@ -23,10 +23,18 @@ body {
...
+    app-view {
+      padding-bottom: 15px;
+    }
+
     &.headless {
         padding-top: 10px;
-        
+
+        app-view {
+          padding-bottom: 0;
```
Comment (ranbena) [inline client/app/assets/less/inc/base.less]: Why is it needed?

Comment (kravets-levko) [reply client/app/assets/less/inc/base.less]: Previously in headless mode padding was removed as well - I just restored that behavior

Comment (ranbena) [reply client/app/assets/less/inc/base.less]: Perhaps it has to do with the iframe version not having a needless vertical scroll or sth

**Comment by kravets-levko** [comment]
Percy caught few more issues with this fix. Need to check once more

**Comment by ranbena** [comment]
I sent you a PR with some fixes

**Comment by kravets-levko** [comment]
@ranbena Thanks a lot!

**Pull request merged**